.grit/patterns/hathora_to_new_ts_sdk.md

```diff
@@ -0,0 +1,391 @@
+# Upgrade Hathora to Dedicated TS SDK
+
+```grit
+engine marzano(0.1)
+language js
+
+function wrapper_key($method) {
+  if ($method <: or {`deleteApp`, `destroyRoomDeprecated`, `destroyRoom`, `suspendRoom`, `deleteBuild`, `suspendRoomDeprecated`}) {
```

Comment:
These big if conditions are not super idiomatic - instead of a function here I think we probably just want a pattern on `$method`.

Reply:
Especially since many of these same methods are duplicated down below in the pattern, I think an inline rewrite would be ideal.